Makefile

```diff
@@ -15,7 +15,7 @@
 
 CFLAGS ?= -O2 -Wall -Wshadow
 
-GIT_VERSION := $(shell git describe --abbrev=6 --dirty --always || date +%F)
+GIT_VERSION := $(shell type git >/dev/null && git describe --abbrev=6 --dirty --always || date +%F)
```

Comment:
Sorry yes - the compile succeeds - this change is just to properly handle the "/bin/sh: git: not found" error. 
Thinking further, command -v is probably a better choice than type -p, as command is POSIX and thus I think it should work in any (sane) shell: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html